dev-support/k8s/tfjob/operator/service.yaml

```diff
@@ -0,0 +1,19 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    prometheus.io/path: /metrics
+    prometheus.io/scrape: "true"
+    prometheus.io/port: "8443"
```

Comment:
This configuration needs users to setup prometheus, right?
It better for us to add comments for this.

Reply:
Annotations is an unstructured key value map stored with a resource that may be set by external tools to store and retrieve arbitrary metadata.  For more info see https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/

So the user doesn't need to setup prometheus.